app/assets/stylesheets/refills/hero.scss

```diff
@@ -26,7 +25,8 @@
     @include clearfix;
     color: black;
     margin: auto;
-    padding: 3.5em;
+    padding-top: 3.0em;
```

Comment:
Properties should be ordered color, margin, padding-bottom, padding-top, text-align<br>`3.0em` should be written without the mantissa as `3em`